### PR TITLE
[DARKNET FRONTEND]Batchnorm added as part of Dense op for running rnn model for next wo…

### PR DESCRIPTION
**Pull request by siju-samuel**
Batchnorm added as part of Dense op for running rnn model for a word prediction model.
This rnn model have batch norm as part of the dense layer, this scenario is not handled in current darknet frontend.

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

**Comment by siju-samuel** [comment]
@zhreshold @Huyuwei @PariksheetPinjari909  Please review this modification.
